.travis.yml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2020 Adobe. All rights reserved.
+# This file is licensed to you under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License. You may obtain a copy
+# of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+# OF ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+language: node_js
+os:
+  - linux
+  - osx
```

Comment:
This should include Windows, as this is an example of an actual developer project that would be built on a developer machines.

**However**, it will not work, as our test framework `aio asset-compute test-worker` needs docker, and docker is not available for neither `osx` nor `windows` in TravisCI. I have the same problem in https://github.com/adobe/aio-cli-plugin-asset-compute/pull/3 where I'm adding unit tests for the command.

If you look at the travis builds here, the tests actually fail or do not complete:
- https://travis-ci.com/github/adobe/asset-compute-example-workers/jobs/338574963
- https://travis-ci.com/github/adobe/asset-compute-example-workers/jobs/338574967 (`docker: command not found`)

Apparently somewhere the non-zero exit code is lost (`cross-os`?), so it ignores it and builds are marked as successful.

So I think we have to limit this to `linux` (in which case the `os` can be removed). Or investigate if windows+docker and mac+docker are possible on CircleCI.

Reply:
@alexkli This should probably a separate ticket then.

Reply:
Ticket https://jira.corp.adobe.com/browse/NUI-555

Reply:
👍